Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/EditingAndDeletingSyncUp.tutorial

```diff
@@ -308,13 +318,12 @@
       only be invoked inside an effect, and so we must use the
       ``ComposableArchitecture/Effect/run(priority:operation:catch:fileID:filePath:line:column:)`` effect, which
       gives us an asynchronous context to execute the work in, as well as a handle on 
-      ``ComposableArchitecture/Send`` for sending actions back into the system.
+      ``ComposableArchitecture/Send`` for sending actions back into the system if needed.
```

Comment:
📓 Here I've added "if needed" because the run action has only the call to the dismiss dependency while before there was also the sending of the delegate action. So now, the send is not really needed, hence the update…